tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_InferredAssetSQLExample.ipynb

```diff
@@ -0,0 +1,1215 @@
+              "  \"class_name\": \"Checkpoint\",\n",
+              "  \"config_version\": 1.0,\n",
+              "  \"evaluation_parameters\": {},\n",
+              "  \"module_name\": \"great_expectations.checkpoint\",\n",
+              "  \"name\": \"my_checkpoint\",\n",
+              "  \"profilers\": [],\n",
+              "  \"runtime_configuration\": {},\n",
+              "  \"validations\": [\n",
+              "    {\n",
+              "      \"batch_request\": {\n",
+              "        \"datasource_name\": \"taxi_multi_batch_sql_datasource\",\n",
+              "        \"data_connector_name\": \"inferred_data_connector_multi_batch_asset\",\n",
+              "        \"data_asset_name\": \"yellow_tripdata_sample_2020\",\n",
+              "        \"data_connector_query\": {\n",
+              "          \"batch_filter_parameters\": {\n",
+              "            \"pickup_datetime\": {\n",
+              "              \"year\": 2020,\n",
+              "              \"month\": 2\n",
+              "            }\n",
+              "          }\n",
+              "        }\n",
+              "      },\n",
+              "      \"expectation_suite_name\": \"example_sql_suite\"\n",
+              "    }\n",
+              "  ]\n",
+              "}"
+            ]
+          },
+          "execution_count": 24,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "checkpoint_config = {\n",
+        "    \"name\": \"my_checkpoint\",\n",
+        "    \"config_version\": 1,\n",
+        "    \"class_name\": \"SimpleCheckpoint\",\n",
+        "    \"validations\": [\n",
+        "        {\n",
+        "            \"batch_request\": single_batch_batch_request_from_multi,\n",
+        "            \"expectation_suite_name\": \"example_sql_suite\",            \n",
+        "        }\n",
+        "    ],\n",
+        "}\n",
+        "data_context.add_checkpoint(**checkpoint_config)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 25,
+      "id": "3269dfba",
+      "metadata": {},
+      "outputs": [
+        {
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "e1f0d2ddeae74e6ab8ae1c4936e870a5",
+              "version_major": 2,
+              "version_minor": 0
+            },
+            "text/plain": [
+              "Calculating Metrics:   0%|          | 0/9 [00:00<?, ?it/s]"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "results = data_context.run_checkpoint(\n",
+        "    checkpoint_name=\"my_checkpoint\"\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 26,
+      "id": "c6234082",
+      "metadata": {},
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "False"
```

Comment:
Shouldn't this be True if we created the expectation based on all of the batches?